examples/report/snapshot_load_compare.py

```diff
@@ -20,6 +20,37 @@ def load_snap(fname: str) -> dict:
         {"arp_table": {"properties": ["!ttl"], "count_change_threshold": 10}},
         {"nics": {"count_change_threshold": 10}},
         {"license": {"properties": ["!serial"]}},
+
+        # {"license": {
```

Comment:
I think we should get rid of the commented lines

Reply:
Yes, once I add the unit tests corresponding to these I will remove them in this PR.